common/assessment/assessment.js

```diff
@@ -18,6 +19,7 @@ const getDeskStudy = R.propEq(keys.deskStudy, true)
 const getCanEditData = R.propEq(keys.canEditData, true)
 const getTablesAccess = R.propOr([], keys.tablesAccess)
 const isStatusChanging = R.pipe(getStatus, R.equals(assessmentStatus.changing))
+const isPanEuropean = R.equals(keys.panEuropean)
```

Comment:
rename this to isTypePanEuropean. you are not passing the assessment here, just a string